### PR TITLE
Enhancement of ptp nic details

### PR DESCRIPTION
**Pull request by bnshr**
This PR enhanced the interface details and collected these information using l2discovery.

### CODE REVIEW

File: test/conformance/ptp/ptp.go [hunk 1]
```diff
@@ -222,27 +222,14 @@ var _ = Describe("[ptp]", func() {
...
-				ptpConfig := testconfig.GlobalConfig
-				if ptpConfig.DiscoveredGrandMasterPtpConfig != nil {
-					printInterface(ptpv1.PtpConfig(*ptpConfig.DiscoveredGrandMasterPtpConfig), mapping)
-				}
-				if ptpConfig.DiscoveredClockUnderTestPtpConfig != nil {
-					printInterface(ptpv1.PtpConfig(*ptpConfig.DiscoveredClockUnderTestPtpConfig), mapping)
-				}
-				if ptpConfig.DiscoveredClockUnderTestSecondaryPtpConfig != nil {
-					printInterface(ptpv1.PtpConfig(*ptpConfig.DiscoveredClockUnderTestSecondaryPtpConfig), mapping)
+				for _, ptpInterface := range ptpInterfacesList {
```
Comment (aneeshkp) [inline test/conformance/ptp/ptp.go:227]: Only writing to the log not to teh report ?

File: vendor/modules.txt [hunk 2]
```diff
@@ -239,10 +239,10 @@ github.com/test-network-function/graphsolver-exports
 # github.com/test-network-function/graphsolver-lib v0.0.0-20220923161551-922faf8697be
 ## explicit; go 1.19
```
Comment (aneeshkp) [inline vendor/modules.txt:240]: we have revert back to go 1.18.
Can you rebase and update the deps

File: test/conformance/ptp/ptp.go (hunk 1 above)
Comment (bnshr) [reply test/conformance/ptp/ptp.go:227]: We decided to merge this PR this way. And then work on the reporting using v1 in some way as a temporary solution.

**Pull request merged**